docs/kafka/metrics-monitoring-kafka/README.adoc

```diff
@@ -86,27 +86,71 @@ As a developer or administrator, you can view metrics in {product-kafka} to visu
 Cluster metrics::
 +
 --
-* `kafka_namespace:haproxy_server_bytes_in_total:rate5m`: Number of incoming bytes per second for the cluster in the last five minutes. This metric represents all the data that producers are sending to topics in the cluster. The maximum incoming byte rate is 30 MB per second.
+* `kafka_namespace:haproxy_server_bytes_in_total:rate5m`: Number of incoming bytes per second for the cluster in the last five minutes. This metric represents all the data that producers are sending to topics in the cluster.
++
+The Kafka instance type determines the maximum incoming byte rate, as shown in the following values:
+
+** 1 MiB per second for a trial instance
+** 50 MiB per second for a production instance with a size of 1 streaming unit
+** 100 MiB per second for a production instance with a size of 2 streaming units
+
+* `kafka_namespace:haproxy_server_bytes_out_total:rate5m`: Number of outgoing bytes per second for the cluster in the last five minutes. This metric represents all the data that producers are receiving from topics in the cluster.
++
+The Kafka instance type determines the maximum outgoing byte rate, as shown in the following values:
+
+** 1 MiB per second for a trial instance
```

Comment:
[Optional] I'm wondering if we want to duplicate that information across (it increases likelyhood of forgetting about changing those). We have limits presented in tabular form here: https://access.redhat.com/articles/5979061

I think it will be better for our business capabilities if we can introduce new sizes in one place and point users to the one size reference article that we always keep updating.

Reply:
> [Optional] I'm wondering if we want to duplicate that information across (it increases likelyhood of forgetting about changing those). We have limits presented in tabular form here: https://access.redhat.com/articles/5979061
> 
> I think it will be better for our business capabilities if we can introduce new sizes in one place and point users to the one size reference article that we always keep updating.

Thanks @wtrocki I think we still need to update limit amounts here.

Reply:
Do not see this comment as comment on your PR. This is more on strategy we should pick for documenting values like that. I always recommend to keep those in one place. 

This values are very meaningful for users that would likely to make purchase decisions based on them. Having them out of date or updated in different timing has business implications etc. I'm ok to get those merged into that doc but if we do that we need to have registry for all documents that mention sizes to update them every time something changes (and things would change for sure)

Reply:
> Do not see this comment as comment on your PR. This is more on strategy we should pick for documenting values like that. I always recommend to keep those in one place.
> 
> This values are very meaningful for users that would likely to make purchase decisions based on them. Having them out of date or updated in different timing has business implications etc. I'm ok to get those merged into that doc but if we do that we need to have registry for all documents that mention sizes to update them every time something changes (and things would change for sure)

Yes, agree it causes more maintenance for us.